types/aws-lambda/package.json

```diff
@@ -2,6 +2,8 @@
     "private": true,
     "name": "@types/aws-lambda",
     "version": "8.10.9999",
+    "nonNpm": true,
```

Comment:
Probably this should be "conflict"?

Reply:
(My thought was that "true" was supposed to fail if there was a package on npm so if it's not failing now that's confusing)

Reply:
No, I was right: https://github.com/DefinitelyTyped/DefinitelyTyped/actions/runs/7937875943/job/21675882216?pr=68667#step:11:57

Reply:
ok sweet, thanks for the quick response! I couldn't find any documentation on this except for the readme which mentions "`nonNpm` is a thing that could be `true`" 🤷 

I'll update it to `conflict`

Reply:
Yes, this needs more docs. will make a note of that so I remember.